src/domain/occurrence/eventOccurrenceForm/EventOccurrenceForm.tsx

```diff
@@ -87,6 +89,31 @@ const EventOccurrenceForm: React.FC<Props & GoToPublishingProps> = ({
     Boolean(initialValues.placeId)
   );
 
+  const getLanguagesSorted = () => {
+    const languagesOrder = sortFavorably(
+      Object.values(EVENT_LANGUAGES).map((language) =>
+        t(`common.languages.${language.toLowerCase()}`)
+      ),
+      [
+        EVENT_LANGUAGES.FI,
+        EVENT_LANGUAGES.SV,
+        EVENT_LANGUAGES.EN,
+      ].map((language) => t(`common.languages.${language.toLowerCase()}`))
+    );
+
+    const languages = [
+      ...Object.values(EVENT_LANGUAGES).map((language) => ({
+        label: t(`common.languages.${language.toLowerCase()}`),
+        value: language,
+      })),
+    ].sort(
```

Comment:
Creating new array and spreading is redundant here, this is enough:

```
 const languages = Object.values(EVENT_LANGUAGES)
      .map((language) => ({
        label: t(`common.languages.${language.toLowerCase()}`),
        value: language,
      }))
      .sort(
        (a, b) =>
          languagesOrder.indexOf(a.label) - languagesOrder.indexOf(b.label)
      );
```

Reply:
Fixed.